src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -294,6 +286,12 @@ export default {
 		showLendAgain() {
 			return this.isLentTo && !this.isLessThan25;
 		},
+		selectedOption() {
+			if (isBetween25And50(this.unreservedAmount) || isLessThan25(this.unreservedAmount)) {
+				return Number(this.unreservedAmount).toFixed();
+			}
+			return '25';
```

Comment:
Should this be a number or should the other return be a string? Is there not an issue returning either a number or string from this computed?

Reply:
Previous value in the watcher was string; there is not an issue returning both.